package.json

```
@@ -0,0 +1,9 @@
+{
```

Missing settings below to run nodemon
```suggestion
{
"scripts": {
    "start": "node index.js",
    "server": "nodemon"
  },
```